homeassistant/components/sensor/linux_battery.py

```diff
@@ -35,45 +35,57 @@
 ATTR_VOLTAGE_MIN_DESIGN = 'voltage_min_design'
 ATTR_VOLTAGE_NOW = 'voltage_now'
 
+ATTR_HEALTH = 'health'
+ATTR_PRESENT = 'present'
+ATTR_STATUS = 'status'
+
 CONF_BATTERY = 'battery'
+CONF_DISTRIBUTION = 'distribution'
 
 DEFAULT_BATTERY = 1
-DEFAULT_NAME = 'Battery'
+DEFAULT_NAME = 'battery'
 DEFAULT_PATH = '/sys/class/power_supply'
+DEFAULT_DISTRIBUTION = 'linux'
 
 ICON = 'mdi:battery'
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_BATTERY, default=DEFAULT_BATTERY): cv.positive_int,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_DISTRIBUTION, default=DEFAULT_DISTRIBUTION): cv.string,
```

Comment:
To make it easier for the users should we limit the options to choose from. E.g. something like this:

```python
DISTRIBUTIONS = ['android', 'linux']
...
vol.Optional(CONF_DISTRIBUTION, default=DEFAULT_DISTRIBUTION): vol.In(DISTRIBUTIONS),
```

Reply:
There are so many distributions of Linux. Some user's distribution may not in the limiting options. 

If `configuration validation failed`, that might confuse them.

Maybe the directory structure of `/sys/class/power_supply/...` is different from some distributions, like Android. 

So this time, only add the Android option. If we find more special cases in the future, we can add them again.